OpenEdXMobile/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -226,20 +211,6 @@ public void showCourseUnitDetail(Fragment fragment, int requestCode, EnrolledCou
         fragment.startActivityForResult(courseDetail, requestCode);
     }
 
-    @Deprecated
-    public void showCourseDashboard(Activity activity, EnrolledCoursesResponse model,
-                                    boolean announcements) {
-        Bundle courseBundle = new Bundle();
-        courseBundle.putSerializable(EXTRA_COURSE_DATA, model);
-        courseBundle.putBoolean(EXTRA_ANNOUNCEMENTS, announcements);
-
-        Intent courseDashboard = new Intent(activity, CourseDashboardActivity.class);
-        courseDashboard.putExtra(EXTRA_BUNDLE, courseBundle);
-        courseDashboard.addFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT);
-        activity.startActivity(courseDashboard);
-
-    }
-
     public void showCourseTabsDashboard(Activity activity, EnrolledCoursesResponse model,
```

Comment:
There are many unused functions in this class that can be removed due to the change in this PR. Please do the needful.